modules/aleph/src/Aleph/AlephMaterial.php

```diff
@@ -127,13 +273,38 @@ public static function loansFromBorInfo(\SimpleXMLElement $xml) {
     $loans = array();
     foreach ($items as $item) {
       $material = new self();
-      $material->setTitle((string) $item->z13->{'z13-title'}[0]);
-      $material->setId((string) $item->z30->{'z30-doc-number'}[0]);
-      $material->setDueDate((string) $item->z36->{'z36-due-date'}[0]);
-      $material->setLoanDate((string) $item->z36->{'z36-loan-date'}[0]);
+      $material->setTitle((string) $item->xpath('z13/z13-title')[0]);
+      $material->setId((string) $item->xpath('z30/z30-doc-number')[0]);
+      $material->setDueDate((string) $item->xpath('z36/z36-due-date')[0]);
+      $material->setLoanDate((string) $item->xpath('z36/z36-loan-date')[0]);
       $loans[] = $material;
     }
     return $loans;
   }
 
+  /**
+   * Create material from item.
+   *
+   * @param $item
+   *
+   * @return \Drupal\aleph\Aleph\AlephMaterial
+   */
+  public static function materialFromItem(\SimpleXMLElement $item) {
```

Comment:
Should we add some example xml-documents to the repo? Or maybe just include examples in the source-code? It is very hard to form an opinion on whether $item->xpath('z30/z30-sub-library')[0]); is the best way to query for the data or not ...